### DISCUSSION TITLE
Enable future bookings in admin-backend

### DISCUSSION THREAD

**Discussion by datengraben** [wielebenwir/commonsbooking#1383] [Questions & Answers]
Wieso sehe ich zukünftige Buchungen eigentlich nicht im Admin-Backend? Das hat sicher einen Grund, den ich einfach nicht kenne.

Hier ist ja eine Vorgehensweise beschrieben die das umgeht, und das Anzeigen der Buchungen im Frontend ermöglicht. https://commonsbooking.org/docs/einstellungen/buchungsliste/

**Comment by hansmorb** [comment]
Was genau meinst du? 
Bei mir klappt das Anzeigen von Buchungen in der Zukunft:
![image](https://github.com/wielebenwir/commonsbooking/assets/6433480/5e31eb9f-9ac6-4d08-ab8b-e16e03cc1355)

**Reply by datengraben** [reply]
@hansmorb Sorry also das heißt ohne besondere Filter oder Einstellung? Sondern default.
